app/src/debug/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,7 @@
+<resources>
+    <!--
+    TODO: Before you run your application, you need a Google Maps API key.
+    -->
```

Comment:
This should be placed in a document somewhere, instead of being embedded in the code. Consider starting/extending a section in the README for how to run your app.